src/pages/scientificServices/pipelines/components/inputs/PipelineFloatInput.tsx

```diff
@@ -18,37 +17,49 @@ export const PipelineFloatInput: React.FC<PipelineFloatInputProps> = ({
   validationError,
   onValidation,
 }) => {
-  const { label, placeholder, helpText, validationRegex } = INPUT_DESCRIPTIONS[input.name] || {};
+  const { name, displayName, description, isRequired, defaultValue, minValue, maxValue } = input;
```

Comment:
i dont know why but i think this kind of expansion is cool

Reply:
destructuring is definitely one of my favorite features of javascript